rxbinding/src/main/java/com/jakewharton/rxbinding/view/ViewKeyEvent.java

```diff
@@ -0,0 +1,31 @@
+package com.jakewharton.rxbinding.view;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.view.KeyEvent;
+import android.view.View;
+
+/**
+ * A key event on a view.
+ * <p>
+ * <strong>Warning:</strong> Instances keep a strong reference to the view. Operators that cache
+ * instances have the potential to leak the associated {@link Context}.
+ */
+public final class ViewKeyEvent extends ViewEvent<View> {
+  private final int keyCode;
+  private final KeyEvent keyEvent;
```

Comment:
The `KeyEvent` contains the key code so I'm wondering if this type is even needed. Should we just return the `KeyEvent` directly in the stream?